src/login.ts

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+// SPDX-FileCopyrightText: 2025 Tim Beckmann <beckmann.tim@fh-swf.de>
+
+import { account } from "./appwrite";
+
+if (checkForSession()) {
+    const email = <HTMLInputElement>document.getElementById("email");
```

Comment:
Non-blocking note:

Congratz on having found an alternative syntax for type assertions that I've never seen before. It's legal in .ts files, but illegal in .tsx files. Probably a legacy syntax.